jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -37,6 +37,8 @@ for indentation.
 $font-family: 'Ubuntu', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $font-family-mono: 'Ubuntu Mono', 'Ubuntu-Mono', 'Bitstream Vera Sans Mono', 'Courier New', Courier, monospace;
 
+$asset-path: 'juju-ui/assets/';
```

Comment:
This is interesting because the asset path in Juju 2.0 is actually different than the root path that the gui is loaded from so we actually have to get that value from juju on load. So this won't work for Juju 2.0 story.